index.js

```diff
@@ -24,9 +24,12 @@ class ServerlessSecretBaker {
       "before:offline:start:end": this.cleanupPackageSecrets.bind(this),
       // For invoke local
       "before:invoke:local:invoke": this.packageSecrets.bind(this),
-      "after:invoke:local:invoke": this.cleanupPackageSecrets.bind(this)
+      "after:invoke:local:invoke": this.cleanupPackageSecrets.bind(this),
     };
 
+    if (!("secret-baker-cleanup" in options)) {
+      options["secret-baker-cleanup"] = true;
```

Comment:
Note.  When passing in the option of `--no-secret-baker-cleanup` the prefix of `--no-` is automatically stripped from the options object and converted to a value of `false`.

Reply:
neat, I imagine that was tricky to figure out what was going on.